src/app/games/trexo/tests/TrexoMove.spec.ts

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable max-lines-per-function */
+import { Coord } from 'src/app/jscaip/Coord';
+import { MGPFallible } from 'src/app/utils/MGPFallible';
+import { EncoderTestUtils } from 'src/app/utils/tests/Encoder.spec';
+import { TrexoFailure } from '../TrexoFailure';
+import { TrexoMinimax } from '../TrexoMinimax';
+import { TrexoMove } from '../TrexoMove';
+import { TrexoRules } from '../TrexoRules';
+import { TrexoState } from '../TrexoState';
+
+describe('TrexoMove', () => {
+
+    it('should refuse to create out of board move (player.zero piece)', () => {
+        // Given two coords of piece, with the zero coord being out of range
+        const zero: Coord = new Coord(-1, 0);
+        const one: Coord = new Coord(0, 0);
+
+        // When trying to pass it as param
```

Comment:
argou ment !

Reply:
puis c'pas "when trying to pass it as an argument" mais "when trying to construct a move with it" plutôt non ?